marimo/_smoke_tests/altair/text_selection.py

```diff
@@ -0,0 +1,159 @@
+
+# /// script
+# requires-python = ">=3.12"
+# dependencies = [
+#     "altair==5.5.0",
+#     "marimo",
+#     "vega-datasets==0.9.0",
+# ]
+# ///
+
+import marimo
+
+__generated_with = "0.13.0"
+app = marimo.App(width="medium")
+
+
+@app.cell
+def _():
+    import marimo as mo
+    return (mo,)
+
+
+@app.cell(hide_code=True)
+def _(mo):
+    mo.md(r"""## Basic Text""")
+    return
+
+
+@app.cell
+def _(mo):
+    from vega_datasets import data
+    import altair as alt
+
+    chart = (
+        alt.Chart(data.cars())
+        .mark_text()
+        .encode(
+            x="Horsepower",
+            y="Miles_per_Gallon",
+            text="Origin",
+        )
+    )
+
+    chart = mo.ui.altair_chart(chart)
+    return alt, chart, data
+
+
+@app.cell
+def _(chart, mo):
+    mo.vstack([chart, chart.value.head()])
+    return
+
+
+@app.cell(hide_code=True)
+def _(mo):
+    mo.md(r"""## Bar chart with text""")
+    return
+
+
+@app.cell(hide_code=True)
+def _(alt, data, mo):
+    _source = data.barley()
+
+    bars = (
+        alt.Chart(_source)
+        .mark_bar()
+        .encode(
+            x=alt.X("sum(yield):Q").stack("zero"),
+            y=alt.Y("variety:N"),
+            color=alt.Color("site"),
+        )
+    )
+
+    text = (
+        alt.Chart(_source)
+        .mark_text(dx=-15, dy=3, color="white")
+        .encode(
+            x=alt.X("sum(yield):Q").stack("zero"),
+            y=alt.Y("variety:N"),
+            detail="site:N",
+            text=alt.Text("sum(yield):Q", format=".1f"),
+        )
+    )
+
+    chart2 = mo.ui.altair_chart(bars + text)
+    chart2
+    return (chart2,)
+
+
+@app.cell
+def _(chart2):
+    chart2.value
+    return
+
+
+@app.cell(hide_code=True)
+def _(mo):
+    mo.md(
+        r"""
+        # Bar chart with double text
+
+        This fails for another reason which is that `month(date)` field gets converted into `month_date` and breaks the backend filtering.
+        """
+    )
+    return
+
+
+@app.cell(hide_code=True)
+def _(alt, data, mo):
```

Comment:
This block actually breaks for me (missing month date)

![image](https://github.com/user-attachments/assets/6ea12b72-5496-4f56-be13-927acc4048cd)


But I don't think it's related- just a filter issue

Reply:
yep, that why it says this above the chart

> This fails for another reason which is that `month(date)` field gets converted into `month_date` and breaks the backend filtering.